axelor-human-resource/src/main/resources/views/TimesheetLine.xml

```diff
@@ -162,6 +208,24 @@
 		<action name="action-timesheet-line-atts-overview-readonly"/>
 	</action-group>
 	
+	<action-attrs name="action-team-task-attrs-default-timesheet-line">
+		<attribute name="value" expr="#{__date__}" for="date"/>
+		<attribute name="value" for="teamTask" expr="eval: _teamTask" if="_teamTask != null"/>
+		<attribute name="value" for="project" expr="eval: _project" if="_project != null"/>
+		<attribute name="value" for="user" expr="eval: _user" if="_user != null"/>
+		<attribute name="value" for="product" expr="eval:__user__.employee?.product"/>
+		<attribute name="readonly" for="teamTask" expr="true"/>
+		<attribute name="readonly" for="project" expr="true"/>
+		<attribute name="value" for="toInvoice" expr="eval: false" if="_teamTaskInvoicing == false"/>
+		<attribute name="value" for="toInvoice" expr="eval: true" if="_teamTaskInvoicing == true &amp;&amp; _invoicingType == 1"/>
+		<attribute name="value" for="toInvoice" expr="eval: false" if="_teamTaskInvoicing == true &amp;&amp; _invoicingType == 2"/>
+	</action-attrs>
+	
+	<action-attrs name="action-team-task-attrs-timesheet-line-find-timesheet">
```

Comment:
check conditions properly
as per this change value get set from the second attribute every time